tools/cli/go-whisk-cli/commands/activation.go

```diff
@@ -224,6 +248,35 @@ var activationResultCmd = &cobra.Command{
         return nil
     },
 }
+//lastFlag(args) retrieves the last activation with flag -l or --last
+//Param: Brings in []strings from args
+//Return: Returns a []string with the latest ID or the original args and any errors
+func lastFlag(args []string) ([]string, error) {
+  //Checks to see if there is an ID sent with the --last
+  //If an ID is given with --last then an error will be thrown
+  if flags.activation.last && len(args) == 0  {
+    options := &whisk.ActivationListOptions {
+      Limit: 1,
+      Skip: 0,
+    }
+    activations, _, err := client.Activations.List(options)
+    if err != nil {  //Checks Activations.List for errors when retrieving latest activaiton
+      whisk.Debug(whisk.DbgError, "client.Activations.List() error for flag --last: %s\n", err)
+      return args, err
+    }
+    if len(activations) == 0 {  //Checks to to see if there are activations
+      whiskErr := whisk.MakeWskError(errors.New("Activation list is empty"), whisk.EXITCODE_ERR_GENERAL, whisk.DISPLAY_MSG, whisk.DISPLAY_USAGE)
+      return args, whiskErr
+    } else {
+      whisk.Debug(whisk.DbgInfo, "Appending most recent activation ID into args\n")
+      args = append(args, activations[0].ActivationID)
+    }
+  } else if flags.activation.last && len(args) == 1 {  //Checks conflict between ID and --last
```

Comment:
A `wsk activation get --last` command may have a `FILTER_FIELD` value.  In this case, the check for a command conflict will fail this command.

In the error message, qualify the `ID` - either use the usage label `ACTIVATION_ID` or just spell it out.. i.e. "activation ID".  Message update: `When specifying an activation ID, do not use the --last flag`